### ISSUE TITLE
Can't parse START TRANSACTION

### ISSUE THREAD

**Issue by vkostyukov** [JSQLParser/JSqlParser#1510]
Here is a quick repl experiment (from version 4.2):

```
scala> import net.sf.jsqlparser.parser.CCJSqlParserUtil
import net.sf.jsqlparser.parser.CCJSqlParserUtil

scala> CCJSqlParserUtil.parse("start transaction")
net.sf.jsqlparser.JSQLParserException: Encountered unexpected token: "start" "START"
    at line 1, column 1.

Was expecting one of:

    "("
    "ALTER"
    "CALL"
    "COMMENT"
    "COMMIT"
    "CREATE"
    "DECLARE"
    "DELETE"
    "DESCRIBE"
    "DROP"
    "EXEC"
    "EXECUTE"
    "EXPLAIN"
    "GRANT"
    "IF"
    "INSERT"
    "MERGE"
    "PURGE"
    "RENAME"
    "REPLACE"
    "RESET"
    "ROLLBACK"
    "SAVEPOINT"
    "SET"
    "SHOW"
    "TRUNCATE"
    "UPDATE"
    "UPSERT"
    "USE"
    "VALUES"
    "WITH"
    <K_SELECT>

  at net.sf.jsqlparser.parser.CCJSqlParserUtil.parseStatement(CCJSqlParserUtil.java:190)
  at net.sf.jsqlparser.parser.CCJSqlParserUtil.parse(CCJSqlParserUtil.java:63)
  at net.sf.jsqlparser.parser.CCJSqlParserUtil.parse(CCJSqlParserUtil.java:38)
  ... 34 elided

```

**Comment by manticore-projects** [comment]
Greetings.

`START TRANSACTION` is not a standard compliant SQL command and has not been implemented (yet) in JSQLParser.
What RDBMS is supporting it please? Can you point me on the syntax documentation please?

**Comment by vkostyukov** [comment]
It's available in MySQL, starting from 5.7, at least: https://dev.mysql.com/doc/refman/5.7/en/commit.html

**Comment by wumpz** [comment]
You are right. But JSqlParser does not pretend to be a complete parser for all the database dialects. It is feature request-driven. 

So PRs are welcome or do you want to fund this new feature. I expect there is more to this, not only this `start transaction` statement.
